app/Jobs/ImportTurboVotePosts.php

```diff
@@ -66,8 +65,12 @@ public function tags()
     public function handle(Rogue $rogue)
     {
         // @TODO: We need to write some tests for this import!
+        info("getting records");
+
         $records = $this->getCSVRecords($this->filepath);
 
+        info("records received");
```

Comment:
This reminds me that we need to set up StyleCI on this project to catch nitpicky style problems (these two `info()` calls should be using single-quoted strings according to our style guide). Let's make a ticket so we don't forget!

Reply:
ah yes! [Here's](https://www.pivotaltracker.com/n/projects/2019429/stories/160462809) the card.